libs/mstable/flatcoin/src/state.ts

```diff
@@ -174,9 +174,17 @@ export const {
               : {
                   orderData: data[3]?.['orderData'] as string,
                   type: data[3]?.['orderType'] as number,
-                  keeperFee: (
-                    data[3]?.['keeperFee'] as BigNumberish
-                  ).toString(),
+                  keeperFee: new BigNumber(
+                    (data[3]?.['keeperFee'] as BigNumberish).toString(),
+                  )
+                    .shiftedBy(-COLLATERAL.decimals)
+                    .toFixed(6),
+                  keeperFeeUsd: new BigNumber(
+                    (data[3]?.['keeperFee'] as BigNumberish).toString(),
+                  )
+                    .shiftedBy(-COLLATERAL.decimals)
+                    .multipliedBy(draft.tokens.collateral.price)
+                    .toFixed(6),
```

Comment:
Can it lead to the issue when `tokens.collateral.price` was not fetched yet and equal to `""` ? 
In that case `keeperFeeUsd` probably will be `NaN` and will be updated only after next refetch in 10-15 secs